engine/flink/management/src/it/scala/pl/touk/nussknacker/engine/management/streaming/FlinkStreamingProcessManagerSpec.scala

```diff
@@ -191,22 +194,14 @@ class FlinkStreamingProcessManagerSpec extends FunSuite with Matchers with Scala
       .withValue("processConfig.signals.topic", ConfigValueFactory.fromAnyRef(signalsTopic))
     val flinkModelData = FlinkStreamingProcessManagerProvider.defaultTypeConfig(configWithSignals).toModelData
 
-    val kafkaClient = new KafkaClient(
-      configWithSignals.getString("processConfig.kafka.kafkaAddress"),
-      configWithSignals.getString("processConfig.kafka.zkAddress"))
-
-    try {
-      val consumer = kafkaClient.createConsumer()
-      flinkModelData.dispatchSignal("removeLockSignal", "test-process", Map("lockId" -> "test-lockId"))
-
-      val readSignals = consumer.consume(signalsTopic).take(1).map(m => new String(m.message(), StandardCharsets.UTF_8)).toList
-      val signalJson = CirceUtil.decodeJsonUnsafe[Json](readSignals.head, "invalid signals").hcursor
-      signalJson.downField("processId").focus shouldBe Some(Json.fromString("test-process"))
-      signalJson.downField("action").downField("type").focus shouldBe Some(Json.fromString("RemoveLock"))
-      signalJson.downField("action").downField("lockId").focus shouldBe Some(Json.fromString("test-lockId"))
-    } finally {
-      kafkaClient.shutdown()
-    }
+    val consumer = kafkaClient.createConsumer()
```

Comment:
Why we needn't `kafkaClient.shutdown()` anymore?

Reply:
kafkaClient is created in StreamingDockerTest, and we shut it down in afterAll method